### PR TITLE
feat: polyfills are no longer required

### PR DESCRIPTION
**Pull request by rflechtner**
## fixes KILTProtocol/ticket#2742

Adapts documentation as polyfills are no longer needed for using the sdk.
This lets us drop the ViteJS example as well, as this is just a run-of-the-mill configuration at this point. 

## How to test:

Mostly textual changes. Also check if I haven't missed any references to the viteJS code examples.

## Checklist:

- [ ] I have verified that the code works
- [ ] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [ ] I have [left the code in a better state](https://deviq.com/principles/boy-scout-rule)
- [ ] I have documented the changes (where applicable)
    * Either PR or Ticket to update [the Docs](https://github.com/KILTprotocol/docs)
    * Link the PR/Ticket here

### CODE REVIEW

**Review by arty-name — APPROVED** [review]
I guess the public documentation PR will also benefit from a read by a native speaker like @Dudleyneedham or @lukeg90

**Comment by rflechtner** [comment]
> I guess the public documentation PR will also benefit from a read by a native speaker like @Dudleyneedham or @lukeg90

I agree, that would be good!

File: docs/develop/01_sdk/04_integrate/02_browser.md [hunk 1]
```diff
@@ -3,7 +3,45 @@ id: howto-integrate-browser
 title: Browser
 ---
 
-Integrating the KILT SDK into a browser application involves "tricking" the browser into believing it is running on a Node server.
-This is needed for the advanced cryptographic primitives to work properly.
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-To simplify the setup, we recommend using the [KILT Distillery](./03_distillery.md) or something like [ViteJS](./04_vitejs.md) to bootstrap a project using the KILT SDK.
+Our javascript SDK (`@kiltprotocol/sdk-js`) is ready to be used in a browser context. For rapid prototyping of simple web apps, we provide a code bundle of the entire SDK which you can add embed in a site by adding the following script tag:
```
Comment (lukeg90) [inline docs/develop/01_sdk/04_integrate/02_browser.md]:
```suggestion
Our JavaScript SDK (`@kiltprotocol/sdk-js`) is ready to be used in a browser context. For rapid prototyping of simple web apps, we provide a code bundle of the entire SDK which you can embed in a site by adding the following script tag:
```

File: docs/develop/01_sdk/04_integrate/02_browser.md [hunk 2]
```diff
@@ -3,7 +3,45 @@ id: howto-integrate-browser
...
+
+```html
+<script src="https://unpkg.com/@kiltprotocol/sdk-js@0.33.1/dist/sdk-js.min.umd.js"></script>
+```
+
+The SDK's functions then become available via a new `kilt` property on the global `window` object.
+
+To get started with your first __React application__ using Kilt, we recommend using either the [KILT Distillery](./03_distillery.md) CLI tool or bootstrapping tools like [Create React App](https://create-react-app.dev) or [ViteJS](https://vitejs.dev) that take away some of the complexity in building and testing a React application.
```
Comment (lukeg90) [inline docs/develop/01_sdk/04_integrate/02_browser.md]:
```suggestion
To get started with your first __React application__ using KILT, we recommend using either the [KILT Distillery](./03_distillery.md) CLI tool or bootstrapping tools like [Create React App](https://create-react-app.dev) or [ViteJS](https://vitejs.dev) that take away some of the complexity in building and testing a React application.
```

Comment (lukeg90) [inline docs/develop/01_sdk/04_integrate/02_browser.md]: I would really think twice about recommending CRA as a modern tool. They don't even mention it in the official [React docs](https://react.dev/) anymore

File: docs/develop/01_sdk/04_integrate/02_browser.md [hunk 3]
```diff
@@ -3,7 +3,45 @@ id: howto-integrate-browser
...
+
+After completing the respective tool's recommended steps to initialize your project, simply add the sdk to its dependencies and you are ready to hack away!
```
Comment (lukeg90) [inline docs/develop/01_sdk/04_integrate/02_browser.md]:
```suggestion
After completing the respective tool's recommended steps to initialize your project, simply add the SDK to your dependencies and you are ready to hack away!
```

File: docs/develop/01_sdk/05_troubleshoot.md [hunk 4]
```diff
@@ -33,8 +33,4 @@ Uncaught SyntaxError: redeclaration of import Buffer
 
 ### Solution
 
-You might use `@polkadot/react-identicon` to display an identicon or another library that conflict with the polyfill provided by our ViteJS configuration.
-There is currently no better solution but to remove the dependency and try to use an alternative.
-
-For `@polkadot/react-identicon`, we suggest to use `@polkadot/ui-shared` and implement your own react identicon.
-You can take inspiration from the [Stakeboard](https://github.com/BTE-Trusted-Entity/stakeboard/blob/8a9713f786a05487daa4bfc394c95b60820c5147/src/components/Identicon/Identicon.tsx).
+Your project might be using polyfills for the NodeJS built-in `Buffer`, which can cause conflicts with some polkadot-js libraries such as `@polkadot/react-identicon`. Try if upgrading the SDK and its dependencies to their latest versions allows you to drop these polyfills from your configuration.
```
Comment (lukeg90) [inline docs/develop/01_sdk/05_troubleshoot.md]:
```suggestion
Your project might be using polyfills for the NodeJS built-in `Buffer`, which can cause conflicts with some polkadot-js libraries such as `@polkadot/react-identicon`. You can try upgrading the SDK and its dependencies to their latest versions. It's possible that upgrading will allow you to drop these polyfills from your configuration.
```

File: docs/develop/01_sdk/04_integrate/02_browser.md (hunk 2 above)
Comment (rflechtner) [reply docs/develop/01_sdk/04_integrate/02_browser.md]: oh okay, didn't know that. So people just use viteJS now? Is there another tool that's used instead?

Comment (rflechtner) [reply docs/develop/01_sdk/04_integrate/02_browser.md]: Should we recommend NextJS instead?

Comment (lukeg90) [reply docs/develop/01_sdk/04_integrate/02_browser.md]: Yeah, Vite is the most popular quick-start option these days. Maybe we could also link to React's recommended frameworks:

https://react.dev/learn/start-a-new-react-project

Comment (rflechtner) [reply docs/develop/01_sdk/04_integrate/02_browser.md]: how's [that](https://github.com/KILTprotocol/docs/pull/257/commits/bbc9ff7de1789258e284f5e7a18163158f82e47e)?

Comment (lukeg90) [reply docs/develop/01_sdk/04_integrate/02_browser.md]: Looks good!

**Pull request merged**